platform/components/docker-gbsyncd-broncos/Dockerfile.j2

```diff
@@ -0,0 +1,40 @@
+FROM docker-config-engine-buster
```

Comment:
Please update base docker from buster to bullseye like https://github.com/Azure/sonic-buildimage/pull/11288.